xbmc/platform/posix/main.cpp

```diff
@@ -41,6 +44,42 @@
 #include "input/linux/LIRC.h"
 #endif
 #include "platform/XbmcContext.h"
+  
+#if defined(TARGET_POSIX)
```

Comment:
this file is located in a folder "posix". this ifdef seems rather pointless to me. in case it is required there is something wrong that needs fixing.